setup/paths.sh

```diff
@@ -38,18 +38,14 @@ else
 fi
 
 # Adding Scan Applications to Path
-#cd $BUILD_HOME/cmsgemos/gempython/tools
-#export PYTHONPATH=$PYTHONPATH:$PWD
+#export "PYTHONPATH=$PYTHONPATH:$BUILD_HOME/cmsgemos/gempython/tools"
```

Comment:
* it was commented in the previous version (according to the diff in this PR)
* also no, the setup script in `cmsgemos` should appropriately set the `PYTHONPATH` for use, if it doesn't (it [does)](https://github.com/cms-gem-daq-project/cmsgemos/blob/generic-amc-release-v3/setup/paths.sh#L67) then open an issue there